.gitlab-ci.yml

```diff
@@ -56,3 +64,22 @@ build-test:
   script:
     - cargo build --release
     - cargo test --release --all
+
+build:
+  stage:                           build
+  <<:                              *docker-env
+  <<:                              *collect-artifacts
+  rules:
+    - if: $CI_PIPELINE_SOURCE == "web"
+    - if: $CI_PIPELINE_SOURCE == "schedule"
+    - if: $CI_COMMIT_REF_NAME == "tags"
+    - if: $CI_COMMIT_REF_NAME =~ /^v[0-9]+\.[0-9]+.*$/        # i.e. v1.0, v2.1rc1
+    - if: $CI_COMMIT_REF_NAME == "master"
+      when: manual
+    - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                   # PRs
+      when: manual
+  script:
+    - cargo build --release
+  after_script:
+    - mkdir -p ./artifacts/cargo-contract/
+    - cp ${CARGO_TARGET_DIR}/release/cargo-contract ./artifacts/cargo-contract/
```

Comment:
Should be `canvas-node` rather than `cargo-contract`?

Reply:
lols, ehehe, let me fix that :sweat_smile: